modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/DefaultDDMStorageAdapter.java

```diff
@@ -34,7 +34,8 @@
  */
 @Component(
 	property = {
-		"ddm.storage.adapter.type=default", "service.ranking:Integer=100"
+		"ddm.storage.adapter.type=default", "ddm.storage.adapter.type=json",
+		"service.ranking:Integer=100"
```

Comment:
@henriquelimadotcom we should not include the `ddm.storage.adapter.type=json` here, we already have the `JSONDDMStorageAdapter`.

As far as I know, the `json` type is deprecated, we should only use the `default` from now on.